content/en/metrics/guide/dynamic_quotas.md

```diff
@@ -0,0 +1,82 @@
+algolia:
+  tags: ['custom metrics']
+is_beta: true
+private: true
+---
+
+## Overview
+
+{{< callout url="#" btn_hidden="true">}}
+  This feature is currently in Preview.
+{{< /callout >}} 
+
+As your business scales, managing observability costs could become unruly quickly. Effective custom metrics governance should increase monitoring efficiency rather than cut visibility entirely. And it should not be a time-consuming, cumbersome process to protect your organization from unintentional billing overages. 
+
+By following Datadog's [Best Practices Guide for Custom Metrics Governance](https://docs.datadoghq.com/metrics/guide/custom_metrics_governance/#monitoring-and-prevention), you can monitor spikes in your custom metrics volumes. With **Datadog's Dynamic Metrics Quotas**, you now additionally have automatic spike remediation -- reducing unintentional metrics usage and thereby reduce costs.
+
+Dynamic Metric Quotas, powered by [Datadog Workflows](https://www.datadoghq.com/product/workflow-automation/), allow you to define a quota / threshold of custom metrics at an account, team or metric name level through a metric monitor on your estimated custom metric usage metrics. If your usage exceeds your threshold of choice, causing the metric monitor to alert, Datadog identifies the culprit, spiking metrics and asks you via Slack notificatinon whether you give us permission to automatically apply Metrics without Limits configurations to reduce the usage and costs of these metrics on your behalf. Datadog's Intelligent query insights provide us a list of the actively queried tags that we use to configure problematic metrics such that you are guaranteed the most cost-optimized configuration that doesn't sacrifice visibility. Dynamic metric quotas help protect your organization from unintentional cardinality and budget overages and relieves your teams from having to play whack-a-mole to remediate metric volume spikes.
+
+## Setup
+
+### Prerequisites
+
+1. A Datadog account with [Workflow Automation][3] enabled
+2. The Datadog [Slack integration][5] installed
+
+### Configure blueprint
+
+#### How to build a per-metric quota
+
+1. Open this [workflow blueprint][8], and click "Create from Blueprint"
+2. Click on the green Monitor tile, and enable the "Automatic triggering" toggle. Also copy the **Mention handle** for this workflow -- you'll use this in Step 5 below
+3. Create a [metric monitor][9]. In the **Define the metric** step, select ```datadog.estimated_usage.metrics.custom.by_metric``` as your metric name and choose the ```SUM BY``` space aggregator (recommendd monitor type - Threshold Alert, but you can also use Change or Anomaly Detection)
+4. Under ***Set alert conditions***, define your quota threshold number
+
+{{< img src="/metrics/guide/dynamic_quotas/automated_mwl_workflow_monitor.png" alt="The Make a decision tiles on the Datadog automated Metrics without Limits™ workflow blueprint" style="width:100%;" >}}
```

Comment:
Can we move this image right underneath the header "how to build a per-metric quota"?